app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v4.rb

```diff
@@ -321,6 +321,24 @@ def uuid_from_href(ems_ref)
       URI(ems_ref).path.split('/').last
     end
 
+    def event_fetcher
+      EventFetcher.new(ext_management_system)
+    end
+
+    class EventFetcher
+      attr_reader :ext_management_system
+
+      def initialize(ems)
+        @ext_management_system = ems
+      end
+
+      def events(query_options)
```

Comment:
@masayag, there are only two options passed here - either :max or :since. Both are the same.